canonical-web-and-design/juju.is
# Update the homepage copy

## Done
Updated the copy from the copy doc.

## QA
- Open the homepage
- Check it matches the [copy doc](https://docs.google.com/document/d/1vn6it-3px_KbplnajPlWPonU6W_xp4JdhE02kNQouZA/edit#)

Fixes  https://github.com/canonical-web-and-design/juju-squad/issues/957
Fixes https://github.com/canonical-web-and-design/juju.is/issues/11
